Taskfile.yml

```diff
@@ -1,15 +1,15 @@
 ---
 version: '3'
 vars:
-  SEMVER: '0.0.1'
+  SEMVER: '0.2.0'
 
 tasks:
   build-go-binary:
     desc: Template task to build a go binary
     dir: action
     cmds:
-      - GOOS={{.OS}} GOARCH=amd64 go build -ldflags="-s -w" -o ../bin/action-{{.OS}}-amd64-{{.SEMVER}}
-      - upx -9 ../bin/action-{{.OS}}-amd64-{{.SEMVER}}
+      - GOOS={{.OS}} GOARCH=amd64 go build -ldflags="-s -w" -o ../bin/firmware-action-{{.OS}}-amd64-{{.SEMVER}}
+      - upx -9 ../bin/firmware-action-{{.OS}}-amd64-{{.SEMVER}} || true
```

Comment:
Can you add a comment here so that it's clear why we ignore the return value of upx?